### ISSUE TITLE
Parser for 'traceroute' not found

### ISSUE THREAD

**Issue by assadniang** [CiscoTestAutomation/genieparser#842]
genie==23.8
pyats==23.8

```
    SDWAN-LAN-SW:
        type: router
        os: iosxe
        #platform: sdwan
        #alias: 'uut'
        credentials:
            default:
                username: cisco
                password: cisco
        connections:
            cli:
                protocol: telnet
                ip: 172.30.7.147
```

```
lan_switch.connect(log_stdout=True)
assert lan_switch.connected == True, f'Failed to reconnect to {lan_switch}'
LOGGER.info(f"Successfully connected to {lan_switch}\n")

traceroute = lan_switch.parse('traceroute 192.168.60.3')
```

genie.libs.parser.utils.common.ParserNotFound: Could not find parser for traceroute 192.168.60.3 under ('iosxe', 'router', None)

**Comment by assadniang** [comment]
Hi @SohanTirpude 

```
NFX350S3BR30#traceroute 192.168.60.3
Type escape sequence to abort.
Tracing the route to 192.168.60.3
VRF info: (vrf in name/id, vrf out name/id)
  1 192.168.30.1 4 msec 8 msec 0 msec
  2 192.168.60.1 8 msec *  0 msec
NFX350S3BR30#
```

The Genie parser shows that '**traceroute**' is available for IOS-XE, but it is not there
https://pubhub.devnetcloud.com/media/genie-feature-browser/docs/#/parsers

Thanks

**Comment by SohanTirpude** [comment]
Hello @assadniang,

No, for your specific command i.e. `traceroute 192.168.60.3` the parser is not there. So, most likely we will have to add support for this command.

Kindly give me some time to check this.

Thank you.

**Comment by SohanTirpude** [comment]
Hello @assadniang,

I have added support for your command and now this is the output I am getting after parsing:
```
{
	"traceroute": {
		"192.168.60.3": {
			"address": "192.168.60.3", 
			"hops": {
				"1": {
					"paths": {
						1: {
							"address": "192.168.30.1", 
							"probe_msec": ["4", "8", "0"]
						}
					}
				}, 
				"2": {
					"paths": {
						1: {
							"address": "192.168.60.1", 
							"probe_msec": ["8", "*", "0"]
						}
					}
				}
			}
		}
	}
}
```

Kindly check and confirm.

**Comment by assadniang** [comment]
Hello @SohanTirpude 

Yes, that looks good

Which version do you expect this will be added?

Thanks

**Comment by SohanTirpude** [comment]
Hello @assadniang,

This will be added in 24.5 version. Once the fix is merged, I will update you back here.

Thank you.

**Comment by SohanTirpude** [comment]
Hello @assadniang,

The PR got merged and the fix will available in the upcoming pyats 24.5 version. Hence, I am closing this ticket for now. If you are facing any other issues, kindly open a new ticket.

Thank you.
